schedule/yast/autoyast/autoyast_btrfs_quota.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: autoyast_btrfs_quota
+description: >
+  AutoYaST installation with BTRFS subvolume quotas. Verifies quota limit on the installed system
+  and if its size corresponds to the configured quota limit during installation.
+vars:
+  AUTOYAST_PREPARE_PROFILE: 1
+  DESKTOP: textmode
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/verify_btrfs_quota
+test_data:
+  btrfs_quota:
+    - path: var
```

Comment:
I wonder if we somehow integrate this into our partitioning structure, as those are parameters for the subvolumes, WDYT? Or do you want to keep this `btrfs_qouta` specific values? Keep in mind that we have a plan to automate interactive configuration of btrfs quotas too.

Reply:
So, do you mean to change the structure of test data to be something like this?
```yaml
    partitions:
      - path: var
        quota: 1.00GiB
      ... to potentially use those data with the partitioning-specific:
        role: raw-volume
        id: bios-boot
```

Reply:
we need some 'subvolumes` probably as the subvolumes belong to a specific partition, I guess that should go somewhere in format/mounting option, so when we do it later there, could be reused.

Reply:
Done. Waiting for VRs to be finished.